Engine/Decoify/manageUsers.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+if(!isset($_SESSION)) 
+{  
+	session_start();
+}
+
+include 'db.php';
+
+function NewUser($Username, $Role, $Email, $Status,$password)
+{
+	$hash = password_hash($password, PASSWORD_DEFAULT);
+	$mysqli = db_connect();
+
+	
+	$createdDate = date("Y-m-d H:i:s");
+
+	$sql = "INSERT INTO Users (Username, `Role`, Email, `Status`, `Timestamp`,`Password`)
```

Comment:
Please use prepare statement

Reply:
This I have done and commit and push the changes but it is not showing in pull request. Also with this I have solve one more error in manageUsersViews.php regarding Username field displaying two time in add form but file is not showing to send pull request